src/routes/console/project-[project]/auth/templates/smsVerificationTemplate.svelte

```diff
@@ -0,0 +1,46 @@
+<script lang="ts">
+    import SmsTemplate from './smsTemplate.svelte';
+    import LocaleOptions from './localeOptions.svelte';
+    import type { Models } from '@appwrite.io/console';
+    import { page } from '$app/stores';
+    import { loadSmsTemplate } from './+page.svelte';
+    import { baseSmsTemplate, smsTemplate } from './strote';
+    import { Id } from '$lib/components';
+    import { addNotification } from '$lib/stores/notifications';
+
+    export let localeCodes: Models.LocaleCode[];
+
+    const projectId = $page.params.project;
+    let locale = 'en';
+    let loading = false;
+
+    async function onLocaleChange() {
+        const timeout = setTimeout(() => {
+            loading = true;
+        }, 1);
+        try {
+            const template = await loadSmsTemplate(projectId, 'verification', locale);
+            clearTimeout(timeout);
+            smsTemplate.set(template);
+            $baseSmsTemplate = { ...$smsTemplate };
+        } catch (error) {
+            clearTimeout(timeout);
+            addNotification({
+                type: 'error',
+                message: error.message
+            });
```

Comment:
In `loadSmsTemplate()`, there's a try/catch that can catch an error if `sdk.forConsole.projects.getSmsTemplate()` fails, but that means `loadSmsTemplate()` would return null/undefined in those cases. That would be problematic, right?

Reply:
mmmmm in what way :thinking: we'd just show a notification right?

Reply:
@ArmanNik, wouldn't we add 2 error notifications then:

1. when `sdk.forConsole.projects.getSmsTemplate()` in `loadSmsTemplate()` throws an error
2. in here because template is `null` or `undefined` when it's expected to not be `null` or `undefined`

Or wait...would `loadSmsTemplate()` just not resolve if there was an error 🧐